conf/modules/flight_plan_guided.xml

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="GUIDED_FlightPlan" dir="flight_plan_guided">
+  <doc>
+    <description>GUIDED_FlightPlan</description>
+  </doc>
+
+  <settings>
+    <dl_settings>
+      <dl_settings NAME="guided_flight_plan">
+        <dl_setting var="nom_flight_alt" min="0.2" step="0.1" max="3." module="flight_plan_guided/flight_plan_guided" shortname="nom_flight_alt" param="NOM_FLIGHT_ALT"/>
```

Comment:
I guess `nom` stands for `nominal` ? I suggest using the full name to make it clear. If it is too long for the GCS settings tab, use the `shortname` attribute.